invenio_pages/records/errors.py

```diff
@@ -12,23 +12,15 @@ class PageNotFoundError(Exception):
     """Page not found exception."""
 
     def __init__(self, identifier):
-        """Constructor."""
-        self.identifier = identifier
-
-    @property
-    def description(self):
-        """Exception's description."""
-        return f"The page identified by {self.identifier} cannot be found."
+        """Initialise error."""
+        super().__init__(f"The page identified by {identifier} cannot be found.")
```

Comment:
question: what is the reason behind this change? I'm not saying is not correct, however it does change the exception's API. Before the message was stored in the `description` property, while now is in the exeption's `args`.

Reply:
I got asked to change it in a previous comment. I used as example the errors.py of invenio_administration.